app/src/main/java/com/asfoundation/wallet/App.java

```diff
@@ -104,17 +107,26 @@ private void initializeRakam() {
       Log.e(TAG, "error: ", e);
     }
 
+    int userLevel = idsRepository.getGamificationLevel();
+    String installerPackage =
+        getPackageManager().getInstallerPackageName(BuildConfig.APPLICATION_ID);
+    if (TextUtils.isEmpty(installerPackage)) installerPackage = "other";
```

Comment:
There is a handling of IllegalArgumentException for the getInstallerPackageName.